porting/first-steps.rst

```diff
@@ -64,7 +64,7 @@ Install the required dependencies from official repository or AUR::
       android-sdk-platform-tools bootimg-tools-git
 
 .. Note::
-    Arch uses python3 as default python, which may cause some errors while building. Using a python virtualenv2 is highly suggested. Please refer to `<https://wiki.archlinux.org/index.php/Python/Virtual_environment>`_ for instructions on setting the Virtual Environment.
```

Comment:
i would say "Arch uses `python3` by default, which may cause build errors". Also capitalize Python when not referring to the package.